src/core/ProgressBar/BaseProgressBar/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { makeStyles, Theme } from '@material-ui/core';
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    width: '12.125rem',
+    borderRadius: '0.125rem',
+  },
+  label: {
+    fontSize: '0.75rem',
+    lineHeight: '0.875rem',
+    marginBottom: theme.spacing(1.625),
+    display: 'block',
+    color: theme.palette.text.primary,
+  },
+  good: {
+    '& .MuiLinearProgress-barColorPrimary': {
+      backgroundColor: theme.palette.success.main,
+    },
+  },
+  warning: {
+    '& .MuiLinearProgress-barColorPrimary': {
+      backgroundColor: theme.palette.warning.main,
+    },
+  },
+  low: {
+    '& .MuiLinearProgress-barColorPrimary': {
+      backgroundColor: theme.palette.error.main,
+    },
+  },
```

Comment:
This would not be required anymore since color is already being passed as a prop